Dumping_script.gm9

```diff
@@ -0,0 +1,309 @@
+# author: zeta , jisagi & Glazed_Belmont 
+
+@start
+set DISPLAY_MODE "PKMN Randomizer script\nTYPE: ______"

```

Comment:
if you merged both scripts you need to change this string to only "PKMN Randomizer script" and skip the type